packages/react/src/components/Table/tableReducer.js

```diff
@@ -368,15 +368,27 @@ export const tableReducer = (state = {}, action) => {
       const data = filteredData || state.data;
       // only update the data and filtered data if deleted
       if (action.payload === 'delete') {
+        const { selectedIds } = state.view.table;
+        const { pagination } = state.view;
+        const totalItems = pagination.totalItems - selectedIds.length;
+        const numberOfPages = totalItems / pagination.pageSize;
```

Comment:
This can lead to number of pages being in decimal form. We should probably round up

Reply:
🤦‍♂️ I definitely didn't check that on Friday. Thanks.